unstructured/partition/lang.py

```diff
@@ -220,7 +220,7 @@ def _convert_to_standard_langcode(lang: str) -> str:
 def detect_languages(
     text: str,
     languages: Optional[List[str]] = ["auto"],
-) -> List[str]:
+) -> List[str] | None:
```

Comment:
I don't think this syntax was supported in python3.8, instead you have to use `Optional[List[str]]`

Reply:
(I could be wrong)